WHY_VIM.md

```diff
@@ -1,88 +0,0 @@
-# Why Vim
```

Comment:
Going to include this more concisely in the README, I think I waffle on to much in separate files